### PR TITLE
Remove basic-card content

### LINKED ISSUE #397: Remove the deprecated basic-card content
The spec still mentions basic-card although basic-card has been deprecated, e.g., 
* "Some payment methods, such as basic-card expect little to no processing by the payment handler which simply returns payment card details in the response"
* "For example, for the basic-card payment method, this object will consist of an object with two fields: one for supportedNetworks, and another for supportedTypes."
* code example of payment handler registration, registering capabilities

As a result, we should remove the deprecated basic-card content.

### PR DESCRIPTION
**Pull request by maxlgu**
closes #397 

The following tasks have been completed:

 * [ ] web platform tests (link)
 * [ ] MDN Docs added (link)

N/A

Implementation commitment:

 * [ ] Safari (link to issue)
 * [ ] Chrome (link to issue)
 * [ ] Firefox (link to issue)
 * [ ] Edge (public signal)

N/A

***
<a href="https://pr-preview.s3.amazonaws.com/maxlgu/payment-handler/pull/398.html" title="Last updated on Sep 29, 2021, 8:35 PM UTC (ed747c2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/payment-handler/398/20a0ca9...maxlgu:ed747c2.html" title="Last updated on Sep 29, 2021, 8:35 PM UTC (ed747c2)">Diff</a>

### CODE REVIEW

**Comment by rsolomakhin** [comment]
Is this WIP or ready for review? 🤣

**Comment by maxlgu** [comment]
Wow, thanks for adding yourself spontaneously.

Yes, the PR is ready for review now.

PTAL!

File: index.html [hunk 1]
```diff
@@ -134,11 +134,9 @@ <h2>
           to the user agent to improve the user experience of selecting payment
           credentials:
           <ul>
-            <li>The <a data-lt="PaymentInstrument.method">method</a> and
-            optional <a data-lt=
-            "PaymentInstrument.capabilities">capabilities</a> inform the user
-            agent decision whether to display this instrument as a candidate
-            for payment.
+            <li>The <a data-lt="PaymentInstrument.method">method</a> inform the
```
Comment (ianbjacobs) [inline index.html]: s/inform/informs

**Review by ianbjacobs — APPROVED** [review]
Hi @maxlgu,
Thanks very much for taking this on! I only have one minor editorial suggestion.

**Comment by maxlgu** [comment]
@rsolomakhin , it's lacking someone who has write access to merge it. Do you have write access? Do you want to review it before I merge it?

**Review by rsolomakhin — APPROVED** [review]

**Comment by rsolomakhin** [comment]
LGTM

**Pull request merged**